modules/appeals_api/app/workers/appeals_api/pdf_submit_job.rb

```diff
@@ -89,20 +96,26 @@ def handle_upload_error(appeal, e)
       log_upload_error(appeal, e)
       appeal.update_status(status: 'error', code: e.code, detail: e.detail)
 
-      if e.code == 'DOC201' || e.code == 'DOC202'
-        notify(
-          {
-            'class' => self.class.name,
-            'args' => [appeal.id, appeal.class.to_s, appeal.created_at.iso8601],
-            'error_class' => e.code,
-            'error_message' => e.detail,
-            'failed_at' => Time.zone.now
-          }
-        )
-      else
-        # allow sidekiq to retry immediately
-        raise
-      end
+      # Get the underlying error from EMMS
+      emms_resp_status = e.detail[/(?<=Upstream|Downstream status: )\d*/]
+
+      # if we can't get the EMMS call's response status,
+      # reraise the exception so sidekiq will retry
+      raise if emms_resp_status.blank?
+
+      # re-raise retryable EMMS errors so sidekick will retry
+      raise if RETRYABLE_EMMS_RESP_STATUS_CODES.include?(emms_resp_status)
+
+      # non retryable error, eat the exception so sidekiq WON'T retry and slack notify
+      notify(
+        {
+          'class' => self.class.name,
+          'args' => [appeal.id, appeal.class.to_s, appeal.created_at.iso8601],
+          'error_class' => e.code,
+          'error_message' => e.detail,
+          'failed_at' => Time.zone.now
+        }
+      )
```

Comment:
At first I wondered if we'd be losing Slack alerts for errors that are retrying, but then I remembered that we have the `retry_limits_for_notification` in this job. 👍